src/cmd/sanity_test/sanity_tests.go

```diff
@@ -198,13 +201,14 @@ func getAllSubFolder(
 			fullFolderName   = parentFolder + "/" + childDisplayName
 		)
 
-		messageCount[fullFolderName], _ = ptr.ValOK(child.GetTotalItemCount())
-
-		// recursively check for subfolders
-		if childFolderCount > 0 {
-			parentFolder := fullFolderName
+		if strings.Contains(fullFolderName, dataFolder) {
```

Comment:
This can trip on partial matches.  Ex:
```go
strings.Contains("quxfoo/barbaz", "foo/bar") // true, but should be false
```
You can use the filters package for more strict matching here.
```suggestion
                // filters.PathEquals may also be what you want?
		if filters.PathContains([]string{dataFolder}).Compare(fullFolderName) {
```